quickevent/app/plugins/Competitors/src/competitorwidget.ui

```diff
@@ -90,6 +90,12 @@
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
+        <property name="buttonSymbols">
+         <enum>QAbstractSpinBox::NoButtons</enum>
+        </property>
+        <property name="singleStep">
+         <number>0</number>
```

Comment:
why is single step disabled?

Reply:
When entering the SI number, I suppose nobody use the single step functionality to edit that field, therefore I disabled the single step to avoid user confusion or errors. 

At first I wanted to transform spinbox into a line edit box, but in the process I encountered numerous dificulties. In the end I found out it is easier to keep the spinbox and just disable the buttons and single step.

Reply:
SpinBox with arrow doing nothing is much more confusing than one with +1 func as usual.

Reply:
no arrows are displaying thanks to `QAbstractSpinBox::NoButtons`